Release-Instructions.md

```diff
@@ -21,3 +23,26 @@ The following steps should be taking by project maintainers if they create a new
     - *carthage archive ZipArchive*
 
 5. Attach archive to the release created in step 1.
+
+# Minizip update
+
+The following steps should be taken by project maintainers when they update minizip files.
+
+1. Source is at https://github.com/nmoinvaz/minizip.
+2. Have cmake:
+`brew install cmake`
+3. Run cmake on minizip repo with our desired configuration:
+`cmake . -DMZ_BRG=ON -DMZ_BZIP2=OFF -DMZ_LZMA=OFF`
```

Comment:
Have you thought about not using MZ_BRG=ON but instead including mz_crypt_apple.c?

Reply:
I tried it, but [CMSDecoder](https://developer.apple.com/documentation/security/cmsdecoder) is only on macOS, not on iOS or tvOS.

Reply:
And don't worry about a minimum of macOS 10.7, because we already restrict to minimum macOS 10.8, as seen in https://github.com/ZipArchive/ZipArchive/blob/master/SSZipArchive.podspec. The reason being that we want modern Objective-C syntax, which requires [subscript for dictionaries](https://developer.apple.com/documentation/foundation/nsdictionary/1415430-objectforkeyedsubscript).